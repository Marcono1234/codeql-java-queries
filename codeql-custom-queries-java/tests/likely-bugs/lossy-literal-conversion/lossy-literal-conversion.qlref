./likely-bugs/lossy-literal-conversion.ql
