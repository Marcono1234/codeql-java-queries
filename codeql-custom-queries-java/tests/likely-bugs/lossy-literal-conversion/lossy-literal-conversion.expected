| Test.java:5:13:5:23 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:5:20:5:23 | -... | this literal |
| Test.java:6:13:6:22 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:6:20:6:22 | 256 | this literal |
| Test.java:7:13:7:24 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:7:20:7:24 | -... | this literal |
| Test.java:8:13:8:23 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:8:20:8:23 | 256L | this literal |
| Test.java:9:13:9:26 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:9:20:9:26 | -... | this literal |
| Test.java:10:13:10:25 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:10:20:10:25 | 255.1f | this literal |
| Test.java:11:13:11:26 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:11:20:11:26 | -... | this literal |
| Test.java:12:13:12:25 | (...)... | Performs lossy conversion of $@ to smaller type byte | Test.java:12:20:12:25 | 255.1d | this literal |
| Test.java:14:13:14:26 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:14:21:14:26 | -... | this literal |
| Test.java:15:13:15:25 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:15:21:15:25 | 65536 | this literal |
| Test.java:16:13:16:26 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:16:21:16:26 | -... | this literal |
| Test.java:17:13:17:26 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:17:21:17:26 | 65536L | this literal |
| Test.java:18:13:18:29 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:18:21:18:29 | -... | this literal |
| Test.java:19:13:19:28 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:19:21:19:28 | 65535.1f | this literal |
| Test.java:20:13:20:29 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:20:21:20:29 | -... | this literal |
| Test.java:21:13:21:28 | (...)... | Performs lossy conversion of $@ to smaller type short | Test.java:21:21:21:28 | 65535.1d | this literal |
| Test.java:26:13:26:21 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:26:20:26:21 | -... | this literal |
| Test.java:27:13:27:24 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:27:20:27:24 | 65536 | this literal |
| Test.java:28:13:28:22 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:28:20:28:22 | -... | this literal |
| Test.java:29:13:29:25 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:29:20:29:25 | 65536L | this literal |
| Test.java:30:13:30:24 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:30:20:30:24 | -... | this literal |
| Test.java:31:13:31:27 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:31:20:31:27 | 65535.1f | this literal |
| Test.java:32:13:32:24 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:32:20:32:24 | -... | this literal |
| Test.java:33:13:33:27 | (...)... | Performs lossy conversion of $@ to smaller type char | Test.java:33:20:33:27 | 65535.1d | this literal |
| Test.java:35:13:35:30 | (...)... | Performs lossy conversion of $@ to smaller type int | Test.java:35:19:35:30 | -... | this literal |
| Test.java:36:13:36:29 | (...)... | Performs lossy conversion of $@ to smaller type int | Test.java:36:19:36:29 | 2147483648L | this literal |
| Test.java:37:13:37:30 | (...)... | Performs lossy conversion of $@ to smaller type int | Test.java:37:19:37:30 | -... | this literal |
| Test.java:38:13:38:29 | (...)... | Performs lossy conversion of $@ to smaller type int | Test.java:38:19:38:29 | 2147483800f | this literal |
| Test.java:39:13:39:32 | (...)... | Performs lossy conversion of $@ to smaller type int | Test.java:39:19:39:32 | -... | this literal |
| Test.java:40:13:40:31 | (...)... | Performs lossy conversion of $@ to smaller type int | Test.java:40:19:40:31 | 2147483647.1d | this literal |
| Test.java:42:13:42:43 | (...)... | Performs lossy conversion of $@ to smaller type long | Test.java:42:20:42:43 | -... | this literal |
| Test.java:43:13:43:42 | (...)... | Performs lossy conversion of $@ to smaller type long | Test.java:43:20:43:42 | 19223372036854775808.0f | this literal |
| Test.java:44:13:44:43 | (...)... | Performs lossy conversion of $@ to smaller type long | Test.java:44:20:44:43 | -... | this literal |
| Test.java:45:13:45:42 | (...)... | Performs lossy conversion of $@ to smaller type long | Test.java:45:20:45:42 | 19223372036854775808.0d | this literal |
| Test.java:47:13:47:34 | (...)... | Performs lossy conversion of $@ to smaller type float | Test.java:47:21:47:34 | -... | this literal |
| Test.java:48:13:48:33 | (...)... | Performs lossy conversion of $@ to smaller type float | Test.java:48:21:48:33 | 3.4028235E40d | this literal |
| Test.java:104:9:104:17 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:104:14:104:17 | -... | this literal |
| Test.java:105:9:105:16 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:105:14:105:16 | 256 | this literal |
| Test.java:106:9:106:18 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:106:14:106:18 | -... | this literal |
| Test.java:107:9:107:17 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:107:14:107:17 | 256L | this literal |
| Test.java:108:9:108:20 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:108:14:108:20 | -... | this literal |
| Test.java:109:9:109:19 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:109:14:109:19 | 255.1f | this literal |
| Test.java:110:9:110:20 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:110:14:110:20 | -... | this literal |
| Test.java:111:9:111:19 | ...+=... | Performs lossy conversion of $@ to smaller type byte | Test.java:111:14:111:19 | 255.1d | this literal |
| Test.java:114:9:114:19 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:114:14:114:19 | -... | this literal |
| Test.java:115:9:115:18 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:115:14:115:18 | 65536 | this literal |
| Test.java:116:9:116:19 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:116:14:116:19 | -... | this literal |
| Test.java:117:9:117:19 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:117:14:117:19 | 65536L | this literal |
| Test.java:118:9:118:22 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:118:14:118:22 | -... | this literal |
| Test.java:119:9:119:21 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:119:14:119:21 | 65535.1f | this literal |
| Test.java:120:9:120:22 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:120:14:120:22 | -... | this literal |
| Test.java:121:9:121:21 | ...+=... | Performs lossy conversion of $@ to smaller type short | Test.java:121:14:121:21 | 65535.1d | this literal |
| Test.java:124:9:124:15 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:124:14:124:15 | -... | this literal |
| Test.java:125:9:125:18 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:125:14:125:18 | 65536 | this literal |
| Test.java:126:9:126:16 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:126:14:126:16 | -... | this literal |
| Test.java:127:9:127:19 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:127:14:127:19 | 65536L | this literal |
| Test.java:128:9:128:18 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:128:14:128:18 | -... | this literal |
| Test.java:129:9:129:21 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:129:14:129:21 | 65535.1f | this literal |
| Test.java:130:9:130:18 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:130:14:130:18 | -... | this literal |
| Test.java:131:9:131:21 | ...+=... | Performs lossy conversion of $@ to smaller type char | Test.java:131:14:131:21 | 65535.1d | this literal |
| Test.java:134:9:134:25 | ...+=... | Performs lossy conversion of $@ to smaller type int | Test.java:134:14:134:25 | -... | this literal |
| Test.java:135:9:135:24 | ...+=... | Performs lossy conversion of $@ to smaller type int | Test.java:135:14:135:24 | 2147483648L | this literal |
| Test.java:136:9:136:25 | ...+=... | Performs lossy conversion of $@ to smaller type int | Test.java:136:14:136:25 | -... | this literal |
| Test.java:137:9:137:24 | ...+=... | Performs lossy conversion of $@ to smaller type int | Test.java:137:14:137:24 | 2147483800f | this literal |
| Test.java:138:9:138:27 | ...+=... | Performs lossy conversion of $@ to smaller type int | Test.java:138:14:138:27 | -... | this literal |
| Test.java:139:9:139:26 | ...+=... | Performs lossy conversion of $@ to smaller type int | Test.java:139:14:139:26 | 2147483647.1d | this literal |
| Test.java:142:9:142:37 | ...+=... | Performs lossy conversion of $@ to smaller type long | Test.java:142:14:142:37 | -... | this literal |
| Test.java:143:9:143:36 | ...+=... | Performs lossy conversion of $@ to smaller type long | Test.java:143:14:143:36 | 19223372036854775808.0f | this literal |
| Test.java:144:9:144:37 | ...+=... | Performs lossy conversion of $@ to smaller type long | Test.java:144:14:144:37 | -... | this literal |
| Test.java:145:9:145:36 | ...+=... | Performs lossy conversion of $@ to smaller type long | Test.java:145:14:145:36 | 19223372036854775808.0d | this literal |
| Test.java:148:9:148:27 | ...+=... | Performs lossy conversion of $@ to smaller type float | Test.java:148:14:148:27 | -... | this literal |
| Test.java:149:9:149:26 | ...+=... | Performs lossy conversion of $@ to smaller type float | Test.java:149:14:149:26 | 3.4028235E40d | this literal |
